Mathlib/MeasureTheory/Integral/Bochner/Set.lean

```diff
@@ -690,6 +690,7 @@ theorem setIntegral_mono_ae_restrict (h : f ≤ᵐ[μ.restrict s] g) :
 theorem setIntegral_mono_ae (h : f ≤ᵐ[μ] g) : ∫ x in s, f x ∂μ ≤ ∫ x in s, g x ∂μ :=
   setIntegral_mono_ae_restrict hf hg (ae_restrict_of_ae h)
 
+@[gcongr high] -- higher priority than `integral_mono`
```

Comment:
This comment makes me a bit nervous. Do we have a principled strategy to avoid a blow-up of priorities/ do we ever need priority `higher`/could we get a cycle of lemmas which want to be higher priority than each other?

Reply:
Perhaps all is fine, there's a clear system and nothing will happen --- if so, can you document that order, please?